pyfixest/did/visualize.py

```diff
@@ -72,26 +93,132 @@ def panelview(
         subsamp = 50,
         title = "Treatment Assignment"
     )
+
+    # Outcome plot
+    panelview(
+        data = df_het,
+        unit = "unit",
+        time = "year",
+        type = "outcome",
+        outcome = "dep_var",
+        treat = "treat",
+        subsamp = 50,
+        title = "Outcome Plot"
+    )
     ```
     """
-    treatment_quilt = data.pivot(index=unit, columns=time, values=treat)
-    treatment_quilt = treatment_quilt.sample(subsamp) if subsamp else treatment_quilt
-    if collapse_to_cohort:
-        treatment_quilt = treatment_quilt.drop_duplicates()
-    if sort_by_timing:
-        treatment_quilt = treatment_quilt.loc[
-            treatment_quilt.sum(axis=1).sort_values().index
-        ]
-    if not ax:
-        f, ax = plt.subplots()
-    cax = ax.matshow(treatment_quilt, cmap="viridis", aspect="auto")
-    f.colorbar(cax) if legend else None
-    ax.set_xlabel(xlab) if xlab else None
-    ax.set_ylabel(ylab) if ylab else None
+    if type == "outcome" and outcome:
```

Comment:
Can we raise an informative error message when a user specifies `type = "outcome"` but does not provide an outcome variable? 

I.e. we could do 

```python
if type == "outcome" and not outcome: 
   raise ValueError("You specified ... but ...")
```
We would then also add a test in `test_error_warnings.py`